deploy/dependencies/admission-webhook/prometheus-rule-validating-webhook.yaml

```diff
@@ -12,18 +12,18 @@ webhooks:
     # NOTE: the caBundle get automatically injected by OLM
     caBundle: Cg==
     service:
-      # NOTE: the name and namespace will be replaced by OLM
+      # NOTE: when changing the service, ensure the same changes are applied
+      # to alertmanager-config-validating-webhook as well
       name: obo-prometheus-operator-admission-webhook
       namespace: operators
       path: /admission-prometheusrules/validate
-      port: 8443
-  failurePolicy: Ignore
+  failurePolicy: Fail
```

Comment:
I wouldn't change the failure policy as part of this PR. 'Fail' means that applying PrometheusRule will be rejected when the admission webhook service is unavailable or unreachable. We haven't found a good solution for this in CMO for instance.